packages/fxa-content-server/webpack.config.js

```diff
@@ -254,4 +246,10 @@ if (ENV === 'development') {
   });
 }
 
+if (ENV === 'production') {
+  webpackConfig.cache = {
+    type: 'filesystem',
+  };
```

Comment:
@dschom just so you are aware of this webpack config.  as it is here it won't help anything in CI, but we can Make It Work at some point.